src/base/static/components/organisms/site-header.js

```diff
@@ -32,7 +33,7 @@ import {
 } from "../../state/ducks/left-sidebar";
 import mq from "../../../../media-queries";
 
-const SiteHeaderWrapper = styled("header")(props => ({
+const SiteHeaderWrapper = styled("div")(props => ({
```

Comment:
Out of curiosity, is it wrong to use the semantic `header` element here? I would think that would provide an accessibility advantage, but I'm not sure.

Reply:
Thanks for catching this! I'll push up a fix to address these issues.

I believe I was debugging the arai issues, and thought the header was causing problems, but it wasn't.

I'll also push up a fix for the nav button in your comment below.